geometry/create-and-edit-geometries/src/main/java/com/esri/samples/create_and_edit_geometries/CreateAndEditGeometriesController.java

```diff
@@ -0,0 +1,440 @@
+    redoButton.disableProperty().bind(geometryEditor.canRedoProperty().not());
+
+    geometryEditor.selectedElementProperty().addListener(((observable, oldValue, newValue) -> {
+      if (newValue != null) {
+        // if an element is selected in the geometry editor
+        // bind the disable property of the delete selected element button to the selected element's can delete property
+        deleteSelectedElementButton.disableProperty().bind(geometryEditor.getSelectedElement().canDeleteProperty().not());
+      } else {
+        // if no element is selected in the geometry editor disable the delete selected element button
+        deleteSelectedElementButton.disableProperty().unbind();
+        deleteSelectedElementButton.setDisable(true);
+      }
+    }));
+
+    // bind the disable property of the stop and save button to the geometry editor's started and can undo properties
+    stopAndSaveButton.disableProperty().bind(Bindings.or(geometryEditor.startedProperty().not(), geometryEditor.canUndoProperty().not()));
+
+    // bind the disable property of the stop and discard button to the geometry editor's started property
+    stopAndDiscardButton.disableProperty().bind(geometryEditor.startedProperty().not());
+
+    // bind the disable property of the delete all geometries button to the geometry editor's started property
+    // and to whether there are any graphics in the graphics overlay
+    deleteAllGeometriesButton.disableProperty().bind(Bindings.or(
+      geometryEditor.startedProperty(), Bindings.createBooleanBinding(() -> graphicsOverlay.getGraphics().isEmpty())));
+
+    // allow user to immediately select and edit graphics
+    mapView.setOnMouseClicked(e -> {
+      if (e.getButton() == MouseButton.PRIMARY && e.isStillSincePress()){
+      graphicsOverlay.clearSelection();
+      Point2D mapViewPoint = new Point2D(e.getX(), e.getY());
+
+      // get graphics near the clicked location
+      ListenableFuture<IdentifyGraphicsOverlayResult> identifyGraphics;
+      identifyGraphics = mapView.identifyGraphicsOverlayAsync(graphicsOverlay, mapViewPoint, 10, false);
+
+      identifyGraphics.addDoneListener(() -> {
+        try {
+          // only select graphic if user not already creating/editing a geometry
+          if (!geometryEditor.isStarted()) {
+            if (!identifyGraphics.get().getGraphics().isEmpty()) {
+
+              // store the selected graphic
+              selectedGraphic = identifyGraphics.get().getGraphics().get(0);
+              selectedGraphic.setSelected(true);
+
+              // hide the selected graphic & start an editing session with a copy of it
+              geometryEditor.start(selectedGraphic.getGeometry());
+              selectedGraphic.setVisible(false);
+            } else {
+              selectedGraphic = null;
+            }
+          }
+        } catch (Exception x) {
+          // on any error, display the stack trace
+          x.printStackTrace();
```

Comment:
```suggestion
        } catch (Exception ex) {
          // on any error, display the stack trace
          ex.printStackTrace();
```
Just better matches other samples